.github/workflows/quick-jobs.yml

```diff
@@ -23,11 +23,15 @@ jobs:
       - name: Set PATH
         # https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#adding-a-system-path
         run: |
-          echo "$HOME/.cabal/bin" >> $GITHUB_PATH
+          echo "$HOME/.cabal/bin:$HOME/.ghcup/bin" >> $GITHUB_PATH
```

Comment:
is not $HOME/.ghcup/bin already in PATH?

Reply:
I probably panicked. Will remove and see.